ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Breakdown.kt

```diff
@@ -60,7 +60,11 @@ fun RBuilder.containerListItem(id: Int, container: FileContainer, sizeType: Meas
 @RFunction
 fun RBuilder.containerListItemHeader(id: Int, container: FileContainer, sizeType: Measurable.SizeType) {
     h2(classes = "accordion-header") {
-        button(classes = "accordion-button collapsed") {
+        var classes = "accordion-button collapsed"
+        if (container.files == null) {
+            classes = "$classes disabled"
```

Comment:
One small thing I noticed while looking at the report - it would probably be better if we can hide the down arrow when the button is disabled. Should be enough to add a CSS rule to set the `.accordion-button::after` element to `display: none` when the button is disabled and remove the `me-3` class from the size if that's the case.

Reply:
Thanks for the suggestion, it looks better now. Already done in 001a451ada5c6e099cb155e7efb0268fa8f9ed0b. It looks like this now:
![Captura de Pantalla 2022-11-29 a las 18 34 37](https://user-images.githubusercontent.com/18336786/204601403-99bc9b2f-2208-483e-8b07-e1eea5d412a4.png)